test/Example.ts

```diff
@@ -11,7 +13,9 @@ afterAll(() => {
 describe('Example', () => {
   describe('run', () => {
     it('should return void when an example does pass typechecking', async () => {
-      const mockSpawnSync = jest.spyOn(child_process, 'spawnSync').mockImplementation(() => ({ status: 0 }))
+      const mockSpawnSync = jest
+        .spyOn(child_process, 'spawnSync')
+        .mockImplementation(() => ({ status: 0 } as SpawnSyncReturns<Buffer>))
```

Comment:
Test failed here after Jest upgrade. I think this is safe to do like this.

Reply:
Same below.

Reply:
I think that this is fine, but it is weird that the test is failing after upgrading jest since the `stderr` property returned by `spawnSync` is a `Buffer | string` ([Node 12 docs](https://nodejs.org/docs/latest-v12.x/api/child_process.html#child_process_child_process_spawnsync_command_args_options)).

Reply:
Tbh, I was also surprised, but did not investigate further. Now I checked again with the clean `node_modules`. Still the same behavior.

Reply:
Gotcha - since either return value is valid, I will go ahead and resolve this conversation for now.